src/sql/workbench/browser/modal/media/modal.css

```diff
@@ -21,17 +21,13 @@
 .modal:not(.flyout-dialog):not(.callout-dialog) .modal-dialog {
 	margin: auto;
 	width: 640px;
-	height: 480px;
+	max-height: 480px;
```

Comment:
with the 'correct' fix, you don't need this change anymore, right? let's keep the old behavior to avoid unexpected impacts.

Reply:
This fix will remain. To be sure, I tested the modal-body div, adding several paragraphs of information. Overflow causes content to scroll inside the modal. This is helpful.